src/sage.c

```diff
@@ -158,10 +195,10 @@ static void sage_per_forest(const int ThisTask, const int forestnr, int *TotGala
     int nfofs_all_snaps[ABSOLUTEMAXSNAPS] = {0};
 
     /* nhalos is meaning-less for consistent-trees until *AFTER* the forest has been loaded */
-    const int64_t nhalos = load_forest(run_params, forestnr, &Halo, forests_info);
+    const int64_t nhalos = load_forest(run_params, forestnr, &Halo, forest_info);
 
     /* /\* need to actually set the nhalos value for CTREES*\/ */
-    /* forests_info->totnhalos_per_forest[forestnr] = nhalos; */
+    /* forest_info->totnhalos_per_forest[forestnr] = nhalos; */
 
 #ifdef PROCESS_LHVT_STYLE
```

Comment:
This is effectively `raise NotImplemented()`. Whenever future us gets bored and gets around to updating this processing...